docs/source/scargo/scargo-new.rst

```diff
@@ -48,7 +48,7 @@ Do not initialize git repository.
     --target
 
 Chose the target on which you would like to build and manage the project ['x86', 'esp32', 'stm32', 'atsam'].
-ESP32 support: Presently following models are supported 'esp32', 'esp32c2', 'esp32c3', 'esp32s2', 'esp32s3'.
+ESP32 support: Presently following models are supported 'esp32'. Specify chip using --chip or use default (esp32c3, esp32s3, ...).
```

Comment:
So which of these is default "(esp32c3, esp32s3, ...)" ? :D

Reply:
fixed